.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-ts-node scripts/lint.ts --staged
+pnpm exec ts-node scripts/lint.ts --staged
```

Comment:
I don't have `ts-node` installed globally, so this hook was failing. `pnpm exec` will use the project's local ts-node.

Did this because I don't think it'll be obvious to contributors why this is happening + we don't expect them to have specific tools installed globally.

Reply:
Indeed thanks!